ZenPacks/zenoss/Import4/service_definition/imp4opentsdb.json

```diff
@@ -0,0 +1,107 @@
+{
+   "servicePath": "/",
+   "serviceDefinition": {
+        "Name": "Imp4OpenTSDB",
+        "Launch": "manual",
+        "Command": "bash -c \"while echo 'Starting ..'; do [ -f /import4/imp4opentsdb.sh ] && /import4/imp4opentsdb.sh || echo 'Startup script not ready ...'; sleep 7; done;\"",
+        "Description": "OpenTSDB environment for import4",
+        "Services": [],
+        "Tags": [
+            "daemon"
+        ],
+        "ImageID": "zenoss/opentsdb:v11",
```

Comment:
Right now, the service that runs the 'Run' (Zope) injects its ImageID as an environment variable into the process, and the process ('zenpack --isntall <foo>') picks this up and uses it if an ImageID is not specified.  What this means is that if you want to use something other than the zenoss/{resmgr,core} image, then you need to specify it by hand.

We could extend serviced to inject another environment variable with a list of all images involved in the deployment, and then have the zenpack code.  Alternatively, we could start tagging our opentsdb image with "latest" every time we push it, and you could use "latest" here.